packages/realm-react/src/useAuthOperation.tsx

```diff
@@ -42,18 +42,17 @@ export function useAuthOperation<Args extends unknown[]>({
         error: undefined,
         operation: operationName,
       });
-      return operation(...args)
-        .then((res) => {
+      return operation(...args).then(
```

Comment:
I thought you didn't want to return a promise here?

Reply:
True...we could just call this.

Reply:
Fixed.